packages/app/src/app/pages/SignIn/ChooseProvider.tsx

```diff
@@ -1,31 +1,24 @@
 import React from 'react';
-import { useAppState, useActions } from 'app/overmind';
+import { Button as MainButton, Stack, Text } from '@codesandbox/components';
 import {
   AppleIcon,
   github as GitHubIcon,
   CodeSandboxIcon,
   GoogleIcon,
 } from '@codesandbox/components/lib/components/Icon/icons';
-import {
-  Element,
-  Stack,
-  Text,
-  Button as MainButton,
-} from '@codesandbox/components';
-import { css } from '@styled-system/css';
+import { useActions, useAppState } from 'app/overmind';
 import history from 'app/utils/history';
 
 import { Button } from './components/Button';
 
-interface ChooseProviderProps {
+type ChooseProviderProps = {
```

Comment:
Interested in why you changed this to a `type`?

Reply:
@tristandubbeld I see that we have been defining props as types rather than interfaces and, even though the differences are not that many, types seem more appropriate as a way to define the props for function components rather than interfaces.